coil-core/src/androidMain/kotlin/coil3/decode/ImageSource.kt

```diff
@@ -15,6 +16,7 @@ class AssetMetadata(
  */
 class ContentMetadata(
     val uri: Uri,
+    val afd: AssetFileDescriptor
```

Comment:
```suggestion
    val assetFileDescriptor: AssetFileDescriptor,
```

Reply:
Should this be optional? Are there situations were an `AssetFileDescriptor` is not available for a URI?

Reply:
Nope, If we can open a InputStream for a ContentUri, there must have a afd as that I/OStream use afd internally

Reply:
Done